.github/workflows/pants.yaml

```diff
@@ -27,13 +27,37 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - uses: actions/cache@v2
-      id: cache
+      id: cache_pants_setup
       with:
         path: |
           ~/.cache/pants/setup
-          ~/.cache/pants/lmdb_store
+        key: pants-setup-${{ runner.os }}-${{ hashFiles('pants.toml') }}
+        restore-keys: |
+          pants-setup-${{ runner.os }}-${{ hashFiles('pants.toml') }}
+          pants-setup-${{ runner.os }}-
+    - uses: actions/cache@v2
+      id: cached_named_caches
+      with:
+        path: |
           ~/.cache/pants/named_caches
```

Comment:
Are there other named caches other than the `pex_root`? Maybe we should be very explicit here about `pex_root`?

Reply:
Could be an opportunity for education in a comment :nerd_face:

Reply:
There could be others, it is a general mechanism.